stream-chat-android-offline/src/main/java/io/getstream/chat/android/livedata/ChatDomainImpl.kt

```diff
@@ -519,6 +529,22 @@ internal class ChatDomainImpl internal constructor(
         return currentUser.id + "-" + UUID.randomUUID().toString()
     }
 
+    private fun addTypingChannel(channelController: ChannelControllerImpl) {
+        val typingEvent: LiveData<Pair<String, List<User>>> = channelController.typing.map { userList ->
+            channelController.channelId to userList
+        }
+
+        _typingChannels.addSource(typingEvent, _typingChannels::postValue)
+    }
+
+    private fun removeTypingChannel(channelController: ChannelControllerImpl) {
+        val typingEvent: LiveData<Pair<String, List<User>>> = channelController.typing.map { userList ->
+            channelController.channelId to userList
+        }
+
+        _typingChannels.removeSource(typingEvent)
+    }
```

Comment:
This is never called, and it removes a source that's created here, within the method. Seems like this wouldn't do practically anything.

Reply:
It can remove a LiveData of a channel depending of our the `equals` and hash code method of LiveData works... But you're right, we never remove channels from activeChannels map, so this will never be used.

Reply:
Since our offline support isn't used by customers without our view models then we could change that type (include there `channelId`) and omit mapping each time when add and remove source.

Reply:
Removed